x/cdp/types/params_test.go

```diff
@@ -783,24 +805,61 @@ func (suite *ParamsTestSuite) TestParamValidation() {
 		{
 			name: "zero debt auction lot",
 			args: args{
-				globalDebtLimit:  types.DefaultGlobalDebt,
-				collateralParams: types.DefaultCollateralParams,
-				debtParam:        types.DefaultDebtParam,
-				surplusThreshold: types.DefaultSurplusThreshold,
-				surplusLot:       types.DefaultSurplusLot,
-				debtThreshold:    types.DefaultDebtThreshold,
-				debtLot:          sdk.ZeroInt(),
-				breaker:          types.DefaultCircuitBreaker,
+				globalDebtLimit:                    types.DefaultGlobalDebt,
+				collateralParams:                   types.DefaultCollateralParams,
+				debtParam:                          types.DefaultDebtParam,
+				surplusThreshold:                   types.DefaultSurplusThreshold,
+				surplusLot:                         types.DefaultSurplusLot,
+				debtThreshold:                      types.DefaultDebtThreshold,
+				debtLot:                            sdk.ZeroInt(),
+				breaker:                            types.DefaultCircuitBreaker,
+				beginBlockerExecutionBlockInterval: types.DefaultBeginBlockerExecutionBlockInterval,
 			},
 			errArgs: errArgs{
 				expectPass: false,
 				contains:   "debt auction lot should be positive",
 			},
 		},
+		{
```

Comment:
Nice, very helpful to have these when refactoring params.  Easy to break validations